packages/react-button/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { Button, ButtonProps } from '../../Button';
+import { Meta } from '@storybook/react';
+
+// TODO: this is here while waiting for react-icons to merge
+import { CalendarIcon } from '../../tmp-icons.stories';
+
+export const Default = (props: ButtonProps) => <Button {...props}>Button</Button>;
```

Comment:
It seems the `Default` story cannot handle controls, is this intentional ?

![image](https://user-images.githubusercontent.com/20744592/128727881-29c3f7d0-7be7-4753-8ddf-eca4c408b165.png)

Reply:
it only works when running Storybook for individual package - I’ve added warning to the PR description